persistence/module.go

```diff
@@ -95,18 +123,31 @@ func (m *persistenceModule) NewRWContext(height int64) (modules.PersistenceRWCon
 	return PostgresContext{
 		Height: height,
 		DB: PostgresDB{
+			conn:       conn,
 			Tx:         tx,
 			Blockstore: m.blockStore,
 		},
 	}, nil
 }
-
-func (m *persistenceModule) NewReadContext(height int64) (modules.PersistenceReadContext, error) {
-	return m.NewRWContext(height)
-	// TODO (Team) this can be completely separate from rw context.
-	// It should access the db directly rather than using transactions
+func (m *persistenceModule) ResetContext() error {
```

Comment:
This appears to only be a test function and not part of the lifecycle? 

Please add TODO to make it a test func only

Reply:
There's nothing I see here stoping this from being at the context level